src/main/resources/db/migration/changelog-202303281903.xml

```diff
@@ -45,6 +45,4 @@
 
     </changeSet>
 
-
```

Comment:
Not a deal-breaker at all, but watch out for small formatting changes like this. If they are important, they should be in a PR dedicated to formatting changes like that.

Reply:
A bad habit I recognize. Did on my own code a lot. I will make an effort to either not touch it or submit a pull when I do.